Dockerfile

```
@@ -0,0 +1,25 @@
+# Copy dependency files
+COPY pyproject.toml ./
+COPY uv.lock* ./
+
+# Install dependencies
+RUN uv sync --frozen
+
+# Copy application code
+COPY . .
+
+# Expose port (Railway will provide PORT env var)
+EXPOSE ${PORT:-8000}
```

Hadolint (2.14.0)

[error] unexpected '}'
expecting '/', a new line followed by the next instruction, or digit

(DL1000)